systemtests/src/test/java/io/enmasse/systemtest/shared/standard/TopicTest.java

```diff
@@ -46,6 +46,9 @@
 import static io.enmasse.systemtest.TestTag.NON_PR;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
```

Comment:
alsoplease do not use asserts from junit4, use junit5 asserts from org.junit.jupiter